src/main/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorBuildStep.groovy

```diff
@@ -104,6 +108,12 @@ class IqPolicyEvaluatorBuildStep
       IqUtil.doFillIqStageItems(jobCredentialsId, job)
     }
 
+    @Override
+    FormValidation doCheckAdvancedProperties(@QueryParameter String advancedProperties) {
+      // TODO validate that the properties are entered into the correct format?
```

Comment:
Will the `TODO` be done in this PR? If not, maybe add another ticket to address this later and label the `TODO` with the ticket number; otherwise, remove the `TODO`.